fix inefficiently packed structs problem

@PotterDai  

We found there is an inefficiently packed struct `defaultMailbox`:
```
type defaultMailbox struct {
	userMailbox     queue
	systemMailbox   *mpsc.Queue
	schedulerStatus int32
	userMessages    int32
	sysMessages     int32
	invoker         MessageInvoker
	dispatcher      Dispatcher
	suspended       int32
	mailboxStats    []Statistics
}
```
This should be changed to following struct, which has smaller size.
```
type defaultMailbox struct {
	userMailbox     queue
	systemMailbox   *mpsc.Queue
	schedulerStatus int32
	userMessages    int32
	sysMessages     int32
        suspended       int32
	invoker         MessageInvoker
	dispatcher      Dispatcher
	mailboxStats    []Statistics
}
```
Firstly, apologize for this problem, which is actually caused by last time I changed type of `suspended` from `int` to `int32`.

Meanwhile, we used this tool [aligncheck](https://gitlab.com/opennota/check) to scan whole project. We found three more inefficient packed struct. Since `remote/protos.pb.go` is marked not to be changed, I'd like to fix the other three stucts in this issue.

```
> $GOPATH/bin/aligncheck ./...
github.com/AsynkronIT/protoactor-go/mailbox: /Users/emily/gopath/src/github.com/AsynkronIT/protoactor-go/mailbox/mailbox.go:41:6: struct defaultMailbox could have size 96 (currently 104)
github.com/AsynkronIT/protoactor-go/protobuf/protoc-gen-csgrain: /Users/emily/gopath/src/github.com/AsynkronIT/protoactor-go/protobuf/protoc-gen-csgrain/proto.go:32:6: struct ProtoMethod could have size 56 (currently 64)
github.com/AsynkronIT/protoactor-go/protobuf/protoc-gen-gograin: /Users/emily/gopath/src/github.com/AsynkronIT/protoactor-go/protobuf/protoc-gen-gograin/proto.go:32:6: struct ProtoMethod could have size 64 (currently 72)
github.com/AsynkronIT/protoactor-go/remote: /Users/emily/gopath/src/github.com/AsynkronIT/protoactor-go/remote/protos.pb.go:83:6: struct MessageEnvelope could have size 56 (currently 64)
```

--- a/mailbox/mailbox.go
+++ b/mailbox/mailbox.go
@@ -44,9 +44,9 @@ type defaultMailbox struct {
 	schedulerStatus int32
 	userMessages    int32
 	sysMessages     int32
+	suspended       int32
 	invoker         MessageInvoker
 	dispatcher      Dispatcher
-	suspended       int32
 	mailboxStats    []Statistics
 }
 

--- a/protobuf/protoc-gen-csgrain/proto.go
+++ b/protobuf/protoc-gen-csgrain/proto.go
@@ -32,10 +32,10 @@ type ProtoService struct {
 type ProtoMethod struct {
 	Name         string
 	PascalName   string
-	InputStream  bool
 	Input        *ProtoMessage
-	OutputStream bool
 	Output       *ProtoMessage
+	InputStream  bool
+	OutputStream bool
 }
 
 // ProtoAst transforms a FileDescriptor to an AST that can be used for code generation

--- a/protobuf/protoc-gen-gograin/proto.go
+++ b/protobuf/protoc-gen-gograin/proto.go
@@ -33,10 +33,10 @@ type ProtoMethod struct {
 	Index        int
 	Name         string
 	PascalName   string
-	InputStream  bool
 	Input        *ProtoMessage
-	OutputStream bool
 	Output       *ProtoMessage
+	InputStream  bool
+	OutputStream bool
 }
 
 // ProtoAst transforms a FileDescriptor to an AST that can be used for code generation

